pyproject.toml

```diff
@@ -76,6 +77,7 @@ tests = [
     "pytest-cov",
     "pytest-watch",
     "tox",
+    "boto3",
 ]
 asyncpg = ["asyncpg"]
```

Comment:
You'll need to declare another extra so that client projects can declare the extra and get the right version of boto installed, something like
`aws_rds_iam` = ["boto3"]`

Reply:
Yes I was doing that ATM :)